lib/9pfs/9pfs_vnops.c

```diff
@@ -246,7 +246,8 @@ static int uk_9pfs_close(struct vnode *vn __unused, struct vfscore_file *file)
 	return 0;
 }
 
-static int uk_9pfs_lookup(struct vnode *dvp, char *name, struct vnode **vpp)
+static int uk_9pfs_lookup(struct vnode *dvp, const char *name,
```

Comment:
> What optimizations do you have in mind? Can you elaborate?
> 
> I assume that the compiler is intelligent enough to see that a variable is not changed within a certain scope. I am asking because `const char * const` is not used frequently. From the coding practice, I agree that explicitly removing mutability where it's not needed is a good idea. However, we are using this nowhere else in the codebase. It is also a bit bulky.

Nope, no optimizations in mind. Was just wondering if there could be a benefit to having more verbosity in such cases. I can agree that generally this is useless and only slightly may help readablity. But, you're right, this is not done anywhere else and needlessly increases the type length.